client/my-sites/importer/site-importer/site-importer-input-pane.jsx

```diff
@@ -184,19 +185,7 @@ class SiteImporterInputPane extends React.Component {
 		const { hostname, pathname } = url.parse(
 			siteURL.startsWith( 'http' ) ? siteURL : 'https://' + siteURL
 		);
-
-		let errorMessage;
-		if ( ! siteURL ) {
-			errorMessage = this.props.translate( 'Please enter a valid URL.' );
-		} else if ( hostname === 'editor.wix.com' || hostname === 'www.wix.com' ) {
-			errorMessage = this.props.translate(
-				"You've entered the URL for the Wix editor, which only you can access. Please enter your site's public URL. It should look like one of the examples below."
-			);
-		} else if ( hostname.indexOf( '.wixsite.com' ) > -1 && pathname === '/' ) {
-			errorMessage = this.props.translate(
-				"You haven't entered the full URL. Please include the part of the URL that comes after wixsite.com. See below for an example."
-			);
-		}
+		const errorMessage = validateImportUrl( siteURL );
```

Comment:
Move url validation logic into a separate file so it can be shared between Settings > Import and the import signup flow.